raiden/messages/metadata.py

```diff
@@ -47,6 +47,9 @@ def _canonical_dict(self) -> dict:
         }
         return {"route": route, "address_metadata": address_metadata}
 
+    def get_metadata(self) -> Optional[Dict[Address, AddressMetadata]]:
+        return self.address_metadata
+
```

Comment:
Nitpick: Maybe add a property called `address_to_metadata` instead. That way accessing it would read more natural in the validation method.

(Of course fixing the attribute name to be consistent would be even better, but that would need coordination with the LC and on the wire format changes :/)